scene/main/node.cpp

```diff
@@ -1149,6 +1173,49 @@ bool Node::is_processing_unhandled_key_input() const {
 	return data.unhandled_key_input;
 }
 
+void Node::set_auto_translate_mode(AutoTranslateMode p_mode) {
+	ERR_THREAD_GUARD
+	if (data.auto_translate_mode == p_mode) {
+		return;
+	}
+
+	if (p_mode == AUTO_TRANSLATE_MODE_INHERIT && data.inside_tree && !data.parent) {
+		ERR_FAIL_MSG("The root node can't be set to Inherit auto translate mode.");
+	}
+
+	data.auto_translate_mode = p_mode;
+	data.is_auto_translating = p_mode != AUTO_TRANSLATE_MODE_DISABLED;
+	data.is_auto_translate_dirty = true;
+
+	propagate_notification(NOTIFICATION_TRANSLATION_CHANGED);
+}
+
+Node::AutoTranslateMode Node::get_auto_translate_mode() const {
```

Comment:
I don't think we include the enum prefix in the same class that defines it usually (might be wrong though, didn't check).
```suggestion
AutoTranslateMode Node::get_auto_translate_mode() const {
```

Reply:
We do. In fact, it raises an error if we don't. :slightly_smiling_face: